.github/workflows/main.yml

```
@@ -311,25 +311,61 @@ jobs:
+  build-wasm:
+    if: startsWith(github.ref, 'refs/tags/')
+    runs-on:
+      group: arc-runners-build
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v4
+        with:
+          ref: ${{ github.event.pull_request && github.head_ref || github.ref_name }}
+
+      - name: Set up Go
+        uses: actions/setup-go@v4
```

actionlint (1.7.4)

the runner of "actions/setup-go@v4" action is too old to run on GitHub Actions. update the action's version to fix this issue

(action)